apps/frontend/src/dashboard/state.ts

```diff
@@ -0,0 +1,25 @@
+import { selector, atom, useRecoilValue } from 'recoil';
+import { DashboardResponse } from '@asap-hub/model';
+import { authorizationState } from '../auth/state';
+import { getDashboard } from './api';
+
+export const fetchDashboardState = selector<DashboardResponse>({
+  key: 'fetchDashboardState',
+  get: async ({ get }) => {
+    get(refreshDashboardState);
+    const dashboard = await getDashboard(get(authorizationState));
+    return dashboard;
```

Comment:
```suggestion
    return getDashboard(get(authorizationState));
```

So not sure exactly whats happening here. I think there might be a problem as get(authorizationState) will return a promise? 

If it works you can just return the promise and make the function not async